scipy/optimize/tests/test__shgo.py

```diff
@@ -624,20 +624,10 @@ def test_12_sobol_inf_cons(self):
         shgo(test1_2.f, test1_2.bounds, n=1, iters=None,
              options=options, sampling_method='sobol')
 
-    def test_13_high_sobol(self):
```

Comment:
Why is this test removed?

Reply:
Because its goal was to check that the underlying code for `Sobol'` is called with dim > 40 as there was 2 implementations. One for < 40 and another for > 40. This is not relevant anymore as all of this is checked directly in our `stats.qmc` submodule.